components/org.wso2.carbon.identity.recovery/src/main/java/org/wso2/carbon/identity/recovery/internal/service/impl/UserAccountRecoveryManager.java

```diff
@@ -925,30 +931,141 @@ public UserRecoveryData getUserRecoveryData(String code, RecoverySteps step) thr
         return recoveryData;
     }
 
+    /**
+     * Get user recovery data using the recovery flow id.
+     *
+     * @param recoveryFlowId Recovery flow id of the user.
+     * @param step Recovery step.
+     * @throws IdentityRecoveryException If an error occurred while validating the recoveryId.
+     */
+    public UserRecoveryData getUserRecoveryDataFromFlowId(String recoveryFlowId, RecoverySteps step)
```

Comment:
returning param is not defined in the method comment

Reply:
Fixed with 4c8c9f64b52364e581f4f4f1890afe0f11577c08